### PR TITLE
[app_dart] Update readiness check to match Dart

### PR DESCRIPTION
**Pull request by CaseyHillers**
This should reduce issues in the daily promotion

### CODE REVIEW

File: app_dart/app.yaml [hunk 1]
```diff
@@ -4,7 +4,12 @@
 
 runtime: custom
 env: flex
-service: default
+readiness_check:
```
Comment (keyonghan) [inline app_dart/app.yaml:7]: Could you please share how Dart does this?

Comment (CaseyHillers) [reply app_dart/app.yaml:9]: https://dart.googlesource.com/dart_ci/+/refs/heads/master/appengine/app.yaml

Comment (keyonghan) [reply app_dart/app.yaml:9]: Thanks.

**Review by keyonghan — APPROVED** [review]
LGTM

**Pull request merged**